app/src/main/res/layout/ongoing_story_card.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.home.continueplaying.OngoingStoryViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="24dp"
+    android:layout_marginTop="8dp"
+    android:layout_marginEnd="24dp"
+    android:layout_marginBottom="8dp"
+    android:elevation="4dp"
+    android:onClick="@{viewModel::clickOnOngoingStoryTile}"
+    android:padding="8dp"
+    app:layout_constraintBottom_toBottomOf="parent"
```

Comment:
We don't need constraints here since this is the constraint layout itself, and it's not contained in a constraint layout.

Reply:
Removed constraints